pkg/controllers/virtualmachineinstance_controller.go

```diff
@@ -18,34 +18,43 @@ package controllers
 
 import (
 	"context"
-
+	"github.com/AlonaKaplan/kubesecondarydns/pkg/zonemgr"
```

Comment:
maybe better to reorder includes here to 3 sections
where our is on the last section

The apierrors one can be in the middle section

Reply:
done